Epam.TestAutomation.Tests/CareersPageTests.cs

```diff
@@ -0,0 +1,41 @@
+using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Web.PageObjects.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Tests;
+
+public class CareersPageTests : BaseTest
+{
+    private CareersPage careersPage;
+
+    [SetUp]
+    public void SetUp()
+    {
+        careersPage = new CareersPage();
+        careersPage.AcceptAllCookies();
+    }
+
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        BrowserFactory.Browser.ClickElement(careersPage.careersButton);
+        BrowserFactory.Browser.ClickElement(careersPage.findYourDreamJobButton);
+        
+        Assert.True(careersPage.isAmericasDisplayed, "Americas element not displayed");
+        Assert.True(careersPage.isEMEADisplayed, "EMEA element not displayed");
+        Assert.True(careersPage.isAPACDisplayed, "APAC not displayed");
+    }
+    
+    [Test]
+    public void HoverOverCareerMenuTest()
+    {
+        BrowserFactory.Browser.GoToUrl(careersPage.jobListingsUrl);
+        Thread.Sleep(1000);
+        action.MoveToElement(BrowserFactory.Browser.FindElement(careersPage.careersButton)).Build().Perform();
```

Comment:
The problem is that you created a property of Action class but don't create the object of Actions class. So it is a reason to the error. You should use Action method from Browser class.